packages/react/src/Suspense.en.mdx

```diff
@@ -7,6 +7,14 @@ title: <Suspense/>
 
 @suspensive/react's `<Suspense/>` will be just Suspense of original React.
 
+### `<Suspense.CSROnly/>`
+
+`<Suspense.CSROnly/>` will return fallback first. After mount, return children only in client. but in server, return fallback only.
```

Comment:
I can't understand this sentence well 😭

after mount, I think both client, server return children not only client👍

If you're going to compare the two(client, server), I'd like to talk about the `Before mount` situation

Reply:
I absolutely agree you, Could you left suggestion code? I need your help